src/app/services/impac-config/impac-config.coffee

```diff
@@ -1,19 +1,92 @@
-@App.service 'ImpacConfigSvc' , ($log, $q, MnoeCurrentUser, MnoeOrganizations) ->
+@App.service 'ImpacConfigSvc' , ($state, $stateParams, $log, $q, MnoeCurrentUser, MnoeOrganizations, ImpacMainSvc, ImpacRoutes, ImpacTheming, IMPAC_CONFIG) ->
   _self = @
 
+  # Keep track of dashboard designer mode
+  _self.dashboardDesigner = null
+
+  # TODO: backend?
```

Comment:
Yes, there is already a default ACL in the backend if i remember well

Reply:
Ok, I'll double check if this is needed.

Reply:
Ah this is purely to control the UI to remove these buttons:
![image](https://user-images.githubusercontent.com/19894/55297980-b6b99300-5476-11e9-9a4c-b57d299b3778.png)

Nothing to do with the actual ACL, I'll update the comment and leave it in the frontend